Dockerfile

```diff
@@ -7,11 +7,11 @@ RUN go get
 
 RUN CGO_ENABLED=0 go build -o /go/bin/app -ldflags="-X 'main.version=`git rev-parse --short HEAD`'"
 
-FROM gcr.io/distroless/static
```

Comment:
Same here, IIRC, this is prefarable for GCP to build from this image, so maybe we should have two separate dockerfile

Reply:
I don't know, as far as I understand it just makes for small images, but alpine is good enough.
And I want it for the health checks in docker-compose, where our users will use the same docker image as us

Reply:
Ok here are some ideas on how to do it better :https://www.mattknight.io/blog/docker-healthchecks-in-distroless-node-js

But let's keep that as a subject for later - for now, I can live with an alpine based image, it's small enough and surely not _that_ much of a security threat.